### PR TITLE
feat: analyze user behavior across different lending protocols

### PR DESCRIPTION
**Pull request by NueloSE**
The following checks have been cover in this analysis:
- [x] Answer required questions from the setup.
- [x] Has been properly documented.
- [x] The analysis provides meaningful outputs.
- [x] All code runs correctly.

### CODE REVIEW

**Comment by lukaspetrasek** [comment]
Hi, please remove the output of the notebook.

**Comment by NueloSE** [comment]
> Hi, please remove the output of the notebook.

@lukaspetrasek removed the output should i also remove the result interpretations?

**Comment by lukaspetrasek** [comment]
> > Hi, please remove the output of the notebook.
> 
> @lukaspetrasek removed the output should i also remove the result interpretations?

No, keep it like this. It's just that with the outputs, it's a little harder to review the ntb.

File: notebooks/Analyze_user_behavior_across_different_lending_protocols.ipynb [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+   "source": [
+    "collateral_df.groupby(['Protocol','Token'])['Total Collateral (USD)'].sum()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d78664c-e8bc-4b00-997a-f31de512ec84",
+   "metadata": {},
+   "source": [
+    "#### Data Visualization"
```
Comment (lukaspetrasek) [inline notebooks/Analyze_user_behavior_across_different_lending_protocols.ipynb:480]: Could you add Venn diagrams to visualize the USD amounts as well?

**Review by lukaspetrasek — CHANGES_REQUESTED** [review]
@NueloSE Reviewed.

**Comment by lukaspetrasek** [comment]
Hi @NueloSE ,

We'll be merging this PR so that it counts towards the OD Hackaton. You will receive a partial reward, but we'll be happy if you open a new PR and finish the task!

**Pull request merged**